packages/component-library/src/_Themes/Brand/BrandTheme.js

```diff
@@ -0,0 +1,118 @@
+// Use with emotion's Global component
+
+/* Add all of this and BRANDING to theme for 2018 */
+export default {
+  /* Base */
+
+  html: {
+    fontSize: "1rem",
+    lineHeight: "1.4",
+    backgroundColor: "#fffefe",
+    fontFamily: "Merriweather",
+    fontWeight: "400",
+    color: "#001732",
+    minHeight: "100%" /* [3] */,
+    WebkitTextSizeAdjust: "100%" /* [4] */,
+    msTextSizeAdjust: "100%" /* [4] */,
+    MozOsxFontSmoothing: "grayscale" /* [5] */,
+    WebkitFontSmoothing: "antialiased" /* [5] */,
+    WebkitOverflowScrolling: "touch",
+    WebkitTapHighlightColor: "rgba(0, 0, 0, 0)"
+  },
+
+  body: {
+    margin: 0,
+    display: "flex",
+    flexDirection: "column"
+  },
+
+  a: {
+    color: "#1e62bd",
+    textDecoration: "none",
+    cursor: "pointer",
+    opacity: "0.9",
+    transition: "all 0.25s ease-in-out",
+
+    ":hover": {
+      color: "#1e62bd",
+      textDecoration: "none",
+      cursor: "pointer",
+      opacity: 1
+    }
+  },
+
+  /* Global Styles from product_design */
+
+  /* Typography */
+  ".Description": {
+    maxWidth: "900px",
+    margin: "0 auto",
+    textAlign: "left",
+    lineHeight: "1.7"
+  },
+
+  ".Title": {
+    fontSize: "3.57rem",
+    lineHeight: "1.2",
+    fontWeight: "300",
+    fontFamily: "'Rubik', sans-serif",
+    marginBottom: "12px"
+  },
+
+  h1: {
+    // fontSize: '36px',
```

Comment:
It looks like this was meant to cascade before. Can I safely pull these out or is it meant for an older browser?

Reply:
You can pull these out, we're relying on `em` support